docsite/rst/intro_inventory.rst

```diff
@@ -231,8 +231,7 @@ SSH connection::
     ansible_ssh_extra_args
       This setting is always appended to the default ssh command line.
     ansible_ssh_pipelining
-      Determines whether or not to use SSH pipelining. This can override the
-      ``pipelining`` setting in ``ansible.cfg``.
+      Determines whether or not to use SSH pipelining. This can override the ``pipelining`` setting in ``ansible.cfg``.
```

Comment:
Shouldn't this line be word-wrapped like the others ?

Your editor probably has a feature, in vim it's `gqq`